### ISSUE TITLE
FEATURE REQUEST - Option to Rebuild Permissions 

### ISSUE THREAD

**Issue by thosvesta** [serghey-rodin/vesta#1222]
Hi,

Please add an option in the GUI to rebuild the permissions of `..../public_html` directory back to its username. Currently, we need to do it under shell `chown -R admin:admin . ` 

Regards,
JH

**Comment by huloza** [comment]
yeah, good suggestion!

**Comment by DaniWinter** [comment]
Good idea, would also be nice if we could rebuild the permissions of the /home/user/conf folders.

**Comment by anton-reutov** [comment]
received
